### PR TITLE
P#69791 Migrate show reference estate setting properly

### PR DESCRIPTION
**Pull request by hungnc89**

### CODE REVIEW

File: js/onoffice-reference-estate-select.js [hunk 1]
```diff
@@ -2,6 +2,12 @@ jQuery(document).ready(function() {
     const valueSelect = 'reference';
```
Comment (LongTrong-exe) [inline js/onoffice-reference-estate-select.js:2]: I think name of the $valueSelect const should be more precise

File: plugin/Installer/DatabaseChanges.php [hunk 2]
```diff
@@ -747,4 +748,14 @@ public function setDataDetailViewAccessControlValue()
 		$pDetailView->setHasDetailView(true);
 		$pDataDetailViewHandler->saveDetailView($pDetailView);
 	}
+
+	public function updateShowReferenceEstate()
```
Comment (LongTrong-exe) [inline plugin/Installer/DatabaseChanges.php:752]: This function name should also be more precise

**Review by LongTrong-exe — DISMISSED** [review]

**Comment by hungnc89** [comment]
- Tester tested
- Video

https://user-images.githubusercontent.com/72601287/165700624-5e6a7423-32b6-4646-bc95-4102d52a19a6.mp4

**Review by jmaas-onoffice — APPROVED** [review]

File: plugin/Installer/DatabaseChanges.php [hunk 3]
```diff
@@ -781,4 +782,14 @@ public function setDetailTemplate()
 		$pDetailView->setTemplate( key( $firstTemplatePath ) );
 		$pDataDetailViewHandler->saveDetailView( $pDetailView );
 	}
+
+	public function updateShowReferenceEstateOfReferenceList()
+	{
+		$prefix = $this->getPrefix();
+		$sql = "UPDATE {$prefix}oo_plugin_listviews
+				SET `show_reference_estate` = 1
+				WHERE `list_type` = 'reference'";
```
Comment (jmaas-onoffice) [inline plugin/Installer/DatabaseChanges.php:791]: This was a bit ambiguous in the requirements, but this should be set for all list types.

The reason is that previously, reference estates were always shown, for all list types. So for backwards compatibility, we need to keep showing reference estates.

**Review by jmaas-onoffice — CHANGES_REQUESTED** [review]

**Review by jmaas-onoffice — APPROVED** [review]

**Review by jmaas-onoffice — APPROVED** [review]

**Pull request merged**